src/__tests__/handleAction-test.js

```diff
@@ -96,6 +95,18 @@ describe('handleAction()', () => {
             counter: 7
           });
       });
+
+      it('should work with non-FSA', () => {
+        const reducer = handleAction(type, (state) => state, defaultState);
+
+        expect(reducer(undefined, {
+          foo: {
+            bar: 'baz'
+          }
+        })).to.deep.equal({
+          counter: 0
+        });
```

Comment:
Is there a chai assertion that we expect this code block not to throw? I think that's the point, and not that the state is reduced to a certain value.